src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,52 @@
 import React from 'react';
-
 import './MovieCard.scss';
+import { Movie } from '../../types/Movie';
+
+type MovieType = {
+  movie: Movie;
+};
+
+export const MovieCard: React.FC<MovieType> = ({ movie }) => {
+  return (
+    <div className="card" data-cy="Movie">
+      <div className="card-image">
+        <figure className="image is-4by3">
+          <img
+            data-cy="MovieImage"
+            src={movie.imgUrl}
+            alt="Film logo"
+          />
+        </figure>
+      </div>
+
+      <div className="card-content">
+        <div className="media">
+          <div className="media-left">
+            <figure className="image is-48x48">
+              <img src="images/imdb-logo.jpeg" alt="imdb" />
+            </figure>
+          </div>
+
+          <div className="media-content">
+            <p className="title is-8" data-cy="MovieTitle">
+              {movie.title}
+            </p>
+          </div>
+        </div>
+
+        <div className="content">
+          <p data-cy="MovieDescription">
+            {movie.description}
+          </p>
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
-);
+          <a
+            href="https://www.imdb.com/title/tt1375666"
+            data-cy="MovieLink"
+          >
```

Comment:
Maybe it's better to use here
```suggestion
          <a
            href={imdbUrl}
            data-cy="MovieLink"
          >
```